hbase-server/src/main/java/org/apache/hadoop/hbase/io/hfile/HFilePrettyPrinter.java

```diff
@@ -593,6 +618,33 @@ private void printMeta(HFile.Reader reader, Map<byte[], byte[]> fileInfo)
     } else {
       out.println(FOUR_SPACES + "Not present");
     }
+    DoublesSketch keySizeSketch =
+      getDoublesSketchFromMetaBlock(reader, HFileWriterImpl.KEYSIZE_SKETCH_KEY_STR);
+    printDoublesSketch(keySizeSketch, "keySize");
+    DoublesSketch valueSizeSketch =
+      getDoublesSketchFromMetaBlock(reader, HFileWriterImpl.VALUESIZE_SKETCH_KEY_STR);
+    printDoublesSketch(valueSizeSketch, "valueSize");
+  }
+
+  private void printDoublesSketch(DoublesSketch sketch, String name) {
+    double [] quantiles = sketch.getQuantiles(NORMALIZED_RANKS);
```

Comment:
Why are we asking for every single quantile?

Let's provide the interquartile range  (25th and 75th quantile), the median, the 95th, and the 99th. That should give a good summary of both the norm and outliers.

Reply:
I like the every quantile. Doesn't cost and user can pick what quantile they want to view.